apps/ledger-live-mobile/src/screens/SyncOnboarding/GenuineCheckFailedDrawer.tsx

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import { BoxedIcon, Button, Flex, Text } from "@ledgerhq/native-ui";
+import { WarningSolidMedium } from "@ledgerhq/native-ui/assets/icons";
+import { useTranslation } from "react-i18next";
+import QueuedDrawer from "../../components/QueuedDrawer";
+import { TrackScreen } from "../../analytics";
+import GenericErrorView from "../../components/GenericErrorView";
+
+export type Props = {
+  isOpen: boolean;
+  onRetry?: () => void;
+  onSkip?: () => void;
+  onClose?: () => void;
+  productName: string;
+  error: Error | null | undefined;
+};
+
+const GenuineCheckFailedDrawer = ({
```

Comment:
Maybe add a comment to explain that there is 2 cases: an `error` is provided and no `error` is provided